{"uuid": "c78839a2-f9f6-41b0-86dc-07dfdf8246b3", "counter": 12, "stdout": "\u001b[0;31mfatal: [192.168.38.8]: FAILED! => {\"ansible_facts\": {\"discovered_interpreter_python\": \"/usr/bin/python3\"}, \"changed\": true, \"cmd\": [\"ip\", \"link\", \"add\", \"ne_in_S1_br\", \"type\", \"bridge\"], \"delta\": \"0:00:00.011346\", \"end\": \"2024-04-22 04:47:06.272132\", \"msg\": \"non-zero return code\", \"rc\": 2, \"start\": \"2024-04-22 04:47:06.260786\", \"stderr\": \"RTNETLINK answers: File exists\", \"stderr_lines\": [\"RTNETLINK answers: File exists\"], \"stdout\": \"\", \"stdout_lines\": []}\u001b[0m", "start_line": 122, "end_line": 123, "runner_ident": "c7610470-191b-493f-959b-0de972d33a09", "event": "runner_on_failed", "pid": 3890063, "created": "2024-04-22T04:47:06.346288", "parent_uuid": "52540038-3809-6845-d7f0-000000000006", "event_data": {"playbook": "/home/vmadm/demo/RapidServe/southbound/deploy_subnet.yaml", "playbook_uuid": "cd21790d-9386-4b17-8a9d-fbe98f6b623a", "play": "Deploy subnet Infrastructure with bridges and Veth Pairs", "play_uuid": "52540038-3809-6845-d7f0-000000000002", "play_pattern": "hostvms", "task": "Create bridges for each subnet", "task_uuid": "52540038-3809-6845-d7f0-000000000006", "task_action": "command", "resolved_action": "ansible.builtin.command", "task_args": "", "task_path": "/home/vmadm/demo/RapidServe/southbound/deploy_subnet.yaml:16", "host": "192.168.38.8", "remote_addr": "192.168.38.8", "res": {"changed": true, "stdout": "", "stderr": "RTNETLINK answers: File exists", "rc": 2, "cmd": ["ip", "link", "add", "ne_in_S1_br", "type", "bridge"], "start": "2024-04-22 04:47:06.260786", "end": "2024-04-22 04:47:06.272132", "delta": "0:00:00.011346", "msg": "non-zero return code", "invocation": {"module_args": {"_raw_params": "ip link add ne_in_S1_br type bridge", "_uses_shell": false, "stdin_add_newline": true, "strip_empty_ends": true, "argv": null, "chdir": null, "executable": null, "creates": null, "removes": null, "stdin": null}}, "stdout_lines": [], "stderr_lines": ["RTNETLINK answers: File exists"], "ansible_facts": {"discovered_interpreter_python": "/usr/bin/python3"}, "_ansible_no_log": null}, "start": "2024-04-22T04:47:05.627774", "end": "2024-04-22T04:47:06.346003", "duration": 0.718229, "ignore_errors": null, "event_loop": null, "uuid": "c78839a2-f9f6-41b0-86dc-07dfdf8246b3"}}